tmt/base.py

```diff
@@ -545,20 +546,72 @@ def show(self):
         if self.opt('verbose'):
             self._sources()
 
+    def _lint_execute(self):
+        # execute is required for a plan
+        execute = self.node.get('execute')
+        if not execute:
+            echo(verdict(0, 'execute step must be defined with "how"'))
+            return False
+
+        how = execute.get('how')
+        if how not in ['detach', 'tmt']:
```

Comment:
Instead of hard-cording it would be better to use `tmt.steps.execute.ExecutePlugin.methods()` to get the list of supported methods. Similar for other steps.

Reply:
nice, thx

Reply:
this does not seem to work :( @psss could be this needs some initialization first @psss ?

Reply:
I'd say the only thing which is needed is to have the plugins initialized, which is done here:
https://github.com/psss/tmt/blob/5c56f875b9bb0c4fd263f02a71255ece6133ce30/tmt/cli.py#L22

Reply:
cool. ty

Reply:
take same steps for discover plugin. Should be resolved @psss. Thanks for the pointers!